examples/xonotic/cloudbuild.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  
+  #
+  # Creates the initial make + docker build platform
+  #
+  - name: "ubuntu"
+    args:
+    - "bash"
+    - "-c"
+    - "echo 'FROM gcr.io/cloud-builders/docker\nRUN apt-get install make\nENTRYPOINT [\"/usr/bin/make\"]' > Dockerfile.build"
+    waitFor: ["-"]
+
+  - name: gcr.io/cloud-builders/docker
+    id: build-make-docker
+    args: ["build", "-f", "Dockerfile.build", "-t", "make-docker", "."]
+
+  # build xonotic image to GCR
+  - name: make-docker
+    id: build
+    dir: "."
+    env:
+      - REPOSITORY=${_REPOSITORY}
+      - IMAGE_TAG=${_IMAGE_TAG}
+      - DOCKER_RUN_ARGS=--network=cloudbuild
+    args: ["-j", "1", "build"]
+  
+  # push xonotic image to GCR
+  - name: "make-docker"
+    id: push
+    waitFor:
+      - build
+    dir: "."
+    env:
+      - REPOSITORY=${_REPOSITORY}
+      - IMAGE_TAG=${_IMAGE_TAG}
```

Comment:
I don't think we need to pass in the IMAGE_TAG, since we already have that defined in the Makefile.

Reply:
Make sense. Removed IMAGE_TAG👍🏻